JiraPS/Public/Remove-JiraVersion.ps1

```diff
@@ -10,60 +11,75 @@
     .EXAMPLE
        Remove-JiraVersion -ID '66596'
        This example removes the Version given.
-    .INPUTS
-       This function does not accept pipeline input pending class creation.
+     .INPUTS
+        [PSJira.Versions]
```

Comment:
In the future please provide the solution if you have one rather than just the issue. I will located the correct name of the object and make the updates